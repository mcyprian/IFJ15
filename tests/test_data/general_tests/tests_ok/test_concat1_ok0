int main()
{
	string a;
	a="kon koen kooas";
	string b;
	b="kooas";
	cout << a << "\n";
	a=concat(a, b);
	cout << a << "\n";

return 0;
}
