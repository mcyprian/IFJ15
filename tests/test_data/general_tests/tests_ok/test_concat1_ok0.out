kon koen kooas
kon koen kooaskooas
